installer/app/src/views/modal/cluster-delete.js.jsx

```diff
@@ -10,8 +10,13 @@ var ClusterDelete = React.createClass({
 					<header>
 						<h2>Delete cluster {this.props.clusterID}?</h2>
 					</header>
-
-					<button type="submit" style={RedBtnCSS}>Delete Cluster</button>
+					<div>
+						<input type="text" placeholder="Type the cluster ID to confirm deletion"
```

Comment:
Make the placeholder just the cluster ID and move the current one into a `<label/>` so it's clear what the user should type.